tests/integration/product_create.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test;
+use Test::More;
+
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+
+remove_all_products();
+wait_dynamic_front();
+my $ua = new_client();
+create_user($ua, {});
+
+my %args_ref = (
+	code => '200000000099',
+	lang => "en",
+	product_name => "Test-75ml",
+	generic_name => "Tester",
+	quantity => "75 ml",
+	link => "https://github.com/openfoodfacts/openfoodfacts-server",
+	expiration_date => "test",
+	ingredients_text => "apple, milk",
+	origin => "france",
+	serving_size => "10g",
+	packaging_text => "no",
+	action => "process",
```

Comment:
So in the arguments to create_product(), we should pass only data that is specific to the product.

The "action" and "type" can be added in create_product()

".submit" is not needed, it's probably something added by the cgi script